### PR TITLE
[feature-discount] Add function

### PR DESCRIPTION
**Pull request by niego**
# Deskripsi PR
**Description:**
Added a new function `calculate_sum` to sum elements in a list. This addresses issue #42.

**Checklist:**
- [x] Code adheres to the coding style.
- [x] Function has a single clear responsibility.
- [x] All edge cases are handled.
- [x] Unit tests are included and passing.

### CODE REVIEW

File: discount_calculation.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+def calculate_discount(price, customer_type):
+    if customer_type == "regular":
```
Comment (abedniego) [inline discount_calculation.py:5]: Untuk process kalkulasi sudah sesuai, tapi tolong dipertimbangkan untuk memvalidasi apabila price parameter mempunyai 0 value. Terimakasih

Comment (abedniego) [reply discount_calculation.py:5]: Ini sudah sesuai sekarang. 

**Checklist:**
- [x] All edge cases are handled.

**Review by abedniego — APPROVED** [review]
